index.d.ts

```diff
@@ -680,6 +697,91 @@ declare global {
+  }
+
+  interface MinimalBlendMixin {
+    readonly opacity?: number
+    readonly blendMode?: BlendMode
+  }
+
+  interface TextSublayerNode {
+    readonly hasMissingFont: boolean
+
+    paragraphIndent: number
+    paragraphSpacing: number
+
+    fontSize: number | PluginAPI['mixed']
+    fontName: FontName | PluginAPI['mixed']
+    textCase: TextCase | PluginAPI['mixed']
+    textDecoration: TextDecoration | PluginAPI['mixed']
+    letterSpacing: LetterSpacing | PluginAPI['mixed']
+    lineHeight: LineHeight | PluginAPI['mixed']
+    hyperlink: HyperlinkTarget | null | PluginAPI['mixed']
+
+    characters: string
+    insertCharacters(start: number, characters: string, useStyle?: "BEFORE" | "AFTER"): void
+    deleteCharacters(start: number, end: number): void
+
+    getRangeFontSize(start: number, end: number): number | PluginAPI['mixed']
+    setRangeFontSize(start: number, end: number, value: number): void
+    getRangeFontName(start: number, end: number): FontName | PluginAPI['mixed']
+    setRangeFontName(start: number, end: number, value: FontName): void
+    getRangeTextCase(start: number, end: number): TextCase | PluginAPI['mixed']
+    setRangeTextCase(start: number, end: number, value: TextCase): void
+    getRangeTextDecoration(start: number, end: number): TextDecoration | PluginAPI['mixed']
+    setRangeTextDecoration(start: number, end: number, value: TextDecoration): void
+    getRangeLetterSpacing(start: number, end: number): LetterSpacing | PluginAPI['mixed']
+    setRangeLetterSpacing(start: number, end: number, value: LetterSpacing): void
+    getRangeLineHeight(start: number, end: number): LineHeight | PluginAPI['mixed']
+    setRangeLineHeight(start: number, end: number, value: LineHeight): void
+    getRangeHyperlink(start: number, end: number): HyperlinkTarget | null | PluginAPI['mixed']
+    setRangeHyperlink(start: number, end: number, value: HyperlinkTarget | null): void
+    getRangeFills(start: number, end: number): Paint[] | PluginAPI['mixed']
+    setRangeFills(start: number, end: number, value: Paint[]): void
+    getRangeTextStyleId(start: number, end: number): string | PluginAPI['mixed']
+    setRangeTextStyleId(start: number, end: number, value: string): void
+    getRangeFillStyleId(start: number, end: number): string | PluginAPI['mixed']
+    setRangeFillStyleId(start: number, end: number, value: string): void
+  }
+
+  interface ConnectorEndpointPosition {
+    position: { x: number, y: number }
+  }
+
+  interface ConnectorEndpointPositionAndEndpointNodeId {
+    position: { x: number, y: number }
+    endpointNodeId: string
+  }
+
+  interface ConnectorEndpointEndpointNodeIdAndMagnet {
+    endpointNodeId: string
+    magnet: 'NONE' | 'AUTO' | 'TOP' | 'LEFT' | 'BOTTOM' | 'RIGHT' 
+  }
+
+  type ConnectorEndpoint = ConnectorEndpointPosition | ConnectorEndpointEndpointNodeIdAndMagnet | ConnectorEndpointPositionAndEndpointNodeId 
```

Comment:
It would be more consistent to put these further up in the file, not in the Mixins section